Disable the build of the po/ directory

Building the po/ directory complains that the scripts in there have
been generated with gettext 0.17, while we use gettext 0.18 in
Buildroot. Since we don't care that much about po files anyway, just
disable the build of this directory.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/Makefile.am
===================================================================
--- a/Makefile.am
+++ b/Makefile.am
@@ -28,7 +28,7 @@
 
 # Add doc back when we have some real content.
 SUBDIRS = config m4 lib libelf libebl libdwfl libdw libcpu libasm backends \
-	  $(PROGS_SUBDIR) po tests
+	  $(PROGS_SUBDIR) tests
 
 EXTRA_DIST = elfutils.spec GPG-KEY NOTES CONTRIBUTING \
 	     COPYING COPYING-GPLV2 COPYING-LGPLV3
